regnety/config/__init__.py

```diff
@@ -0,0 +1,6 @@
+import os
+os.environ['TF_CPP_MIN_LOG_LEVEL'] = '3' 
+
+
+from regnety.regnety.config.config import get_model_config, ALLOWED_FLOPS
```

Comment:
Do you have a script for counting flops? If you used an external utility do include an example in a script somewhere in the repository to make things transparent.

Reply:
No I don't have one currently. I'll work that out shortly.